<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resultado</class>
 <widget class="QMainWindow" name="Resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mensagens Ocultas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Imagem_principal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salvar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar Imagem</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>211</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titulo_procurar">
       <property name="text">
        <string>Procurar mensagem Oculta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procurar">
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titulo_inserir">
       <property name="text">
        <string>Inserir mensagem Oculta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="Input_text">
       <property name="placeholderText">
        <string>Apenas 6 caracteres (n√∫meros , letras maiusculas e simbolos especiais).</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enviar_msg">
       <property name="text">
        <string>Inserir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Outpu_mensagem">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalvar_Imagem">
   <property name="text">
    <string>Salvar Imagem</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
